### PR TITLE
Fixes non dense, non air-proof indestructible walls.

### PR DESCRIPTION
**Pull request by bear1ake**
Makes centcom walls dense.

### CODE REVIEW

**Comment by vista-** [comment]
Uh, `density = 0` should make them non-dense...?
Also, I don't see /turf/wall being used anywhere on the map, so if this is a working fix with `density = 1`, it's fixing something "deeper" in the object inheritance hierarchy.

**Comment by optimumtact** [comment]
This PR made me dense

**Comment by kyrahabattoir** [comment]
turf/indestructible is everywhere now, does that fix the gas leaking out of centcomm?

File: code/game/turfs/turf.dm [hunk 1]
```diff
@@ -261,6 +261,7 @@
 /turf/indestructible
 	name = "wall"
 	icon = 'icons/turf/walls.dmi'
+	density = 0
```
Comment (xxalpha) [inline code/game/turfs/turf.dm]: This is opposite of dense.

Comment (Cheridan) [inline code/game/turfs/turf.dm]: u tried

Comment (CorruptComputer) [inline code/game/turfs/turf.dm]: you tried

**Comment by Cheridan** [comment]
lmao don't close it, just fix it and push.

**Comment by bear1ake** [comment]
Lol, i'm tried :D

**Pull request merged**